### DISCUSSION TITLE
绳网的UI做的实在是太大了

### DISCUSSION THREAD

**Discussion by HKLHaoBin** [share121/inter-knot#2612] [其他]
![Clip_2024-08-18_00-19-05](https://github.com/user-attachments/assets/660e609a-8ce5-49a8-9994-1d2e1886e67c)
**这个更为合适：**
![Clip_2024-08-18_00-16-37](https://github.com/user-attachments/assets/6f3bf73c-6c5f-4da2-b27d-cf9272eddb82)

**Comment by HaChi0504** [comment]
其實你可以調整縮放就能變這樣了

**Comment by 1luik** [comment]
你这个用ctrl+鼠标滚轮调整缩放（大部分浏览器）
